numpy/core/src/multiarray/calculation.c

```diff
@@ -1211,7 +1211,7 @@ PyArray_Conjugate(PyArrayObject *self, PyArrayObject *out)
                                                 n_ops.conjugate);
         }
         else {
-            return PyArray_GenericBinaryFunction(self,
+            return PyArray_GenericBinaryFunction((PyObject *)self,
```

Comment:
We never test this function with an `out` argument?

Reply:
Strange... I added a small test, this isn't even documented and positional only (that `arr.conjugate(out)` works).